Include/cpython/dictobject.h

```diff
@@ -83,3 +83,6 @@ typedef struct {
 
 PyAPI_FUNC(PyObject *) _PyDictView_New(PyObject *, PyTypeObject *);
 PyAPI_FUNC(PyObject *) _PyDictView_Intersect(PyObject* self, PyObject *other);
+
+PyAPI_FUNC(int) _PyObject_VisitManagedDict(PyObject *self, visitproc visit, void *arg);
```

Comment:
I think this is quite a bit weird: these functions receive an object and then visit/clear the dict. This is not an API for dictionary objects, it's an API for instances so I think it should not be here but in `object.h` or some other header.

Looking at these functions, at first sight, it seems that "self" is the dictionary and not the oject.